calculator.js

```diff
@@ -1,5 +1,73 @@
 const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
+  num1: -3,
+  num2: 4,
+  operation: '/',
+};
+
+
+// when Object is empty
+const objectCantBeEmpty = function(input) {
+  if (Object.keys(input).length === 0) {
+    console.log('Your input is empty! Please try again.');
+    return true;
+  }
```

Comment:
Instead of returning a boolean value here you should just `throw` an exception.

```suggestion
  if (Object.keys(input).length === 0) {
    throw new TypeError('Your input is empty! Please try again.');
  }
```